brian2tools/mdexport/mdexporter.py

```diff
@@ -36,9 +36,13 @@ def build(self, direct_call=True, debug=False, expander=None,
             particular filename. When empty string '' is passed the user file
             name would be taken
 
+        Additionalformat : str and list of str
```

Comment:
The type should be "str *or* list of str". And this should mention the "all" option, as well

Reply:
and available format